front/app/containers/ProjectsShowPage/Modal.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useEffect } from 'react';
+
+// hooks
+import useProject from 'hooks/useProject';
+import usePhases from 'hooks/usePhases';
+import { useSearchParams } from 'react-router-dom';
+
+// styling
+import rocket from 'assets/img/rocket.png';
+
+// components
+import { Box, Title, Image } from '@citizenlab/cl2-component-library';
+import Modal from 'components/UI/Modal';
+
+// utils
+import { isNilOrError } from 'utils/helperUtils';
+import { getMethodConfig, getPhase } from 'utils/participationMethodUtils';
+import { getCurrentPhase } from 'services/phases';
+
+// typings
+import { ParticipationMethod } from 'services/participationContexts';
+
+interface Props {
+  projectId: string;
+}
+
+const _Modal = ({ projectId }: Props) => {
```

Comment:
Nice to put this in a separate component. Perhaps we should name it something else (SuccessModal? or maybe there's a better name), we don't really have a convention of adding `_` to our component names when they conflict.